smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/StructuredMemberWriter.java

```diff
@@ -65,6 +70,140 @@ void writeMembers(TypeScriptWriter writer, Shape shape) {
         }
     }
 
+    /**
+     * Recursively writes filterSensitiveLog for CollectionShape.
+     */
+    void writeCollectionFilterSensitiveLog(TypeScriptWriter writer, MemberShape collectionMember) {
```

Comment:
The `writeCollection` and `writeMap` variants should be private for now.

Reply:
Made them private in https://github.com/awslabs/smithy-typescript/pull/170/commits/ba5d2c96c7eb3abecac37fea130a8f9f6948f4f8